File: Java/SistemaLibreria/src/sistemalibreria/Libro.java
package sistemalibreria;

import java.util.*;
public class Libro 
{
    Scanner leer = new Scanner(System.in);
    Random aleatorio = new Random();
    
    private int id;
    private String tittle;
    private String autor;
    private boolean rentado;
    
   public Libro( String tittle, String autor, int id, boolean rentado)
   {
       this.id = aleatorio.nextInt(1000)+1;
       this.tittle = tittle;
       this.autor = autor;
       this.rentado = false;
   }


    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTittle() {
        return tittle;
    }

    public void setTittle(String tittle) {
        this.tittle = tittle;
    }

    public String getAutor() {
        return autor;
    }

    public void setAutor(String autor) {
        this.autor = autor;
    }

    public boolean isRentado() {
        return rentado;
    }

    public void setRentado(boolean rentado) {
        this.rentado = rentado;
    }
   
    public void rentar()
    {
        this.rentado = true;
    }
    
    
}

File: Python/SistemaLibreria/Libro.py

import random

class Libro:
    __id = 0
    __tittle = ""
    __autor = ""
    __rentado = False

    def __init__(self, tittle, autor, id, rentado ):
        self.__id = random.randint(1,1000)
        self.__tittle = tittle
        self.__autor = autor
        self.__rentado = False
    
    def getId(self):
        return self.__id
    
    def setId(self, id):
        self.__id = id
    
    def getTittle(self):
        return self.__tittle
    
    def setTittle(self, tittle):
        self.__tittle = tittle
    
    def getAutor(self):
        return self.__autor
    
    def setAutor(self, autor):
        self.__autor = autor

    def getRentado(self):
        return self.__rentado
    
    def setRentado(self, rentado):
        self.__rentado = rentado
    
    def rentar(self):
        self.__rentado = True